sklearn/tests/test_multiclass.py

```diff
@@ -239,7 +239,7 @@ def test_ovr_multiclass():
         clf = OneVsRestClassifier(base_clf).fit(X, y)
         assert_equal(set(clf.classes_), classes)
         y_pred = clf.predict(np.array([[0, 0, 4]]))[0]
-        assert_equal(set(y_pred), set("eggs"))
+        assert_equal(set([y_pred]), {"eggs"})
```

Comment:
What's happening here? Why two different solutions for two sets? Maybe ``{y_pred}``?